.circleci/config.yml

```diff
@@ -53,12 +59,10 @@ jobs:
           command: |
             . venv/bin/activate
             cd python
-            python3 -m pytest ml4ir/applications/classification/tests --html=pytest_report.html
+            python3 -m pytest ml4ir/applications/classification/tests --junitxml=test_results/results.xml
           no_output_timeout: 45m  # Time for running the tests before we timeout
-      - store_artifacts:
-          path: python/classification/pytest_report.html
       - store_test_results:
-          path: python/classification/pytest_report.html
+          path: python/test_results
```

Comment:
do we need to make sure this is in .gitignore?

Reply:
That makes sense. This will only leave inside the Docker container and not clutter user's workspace though.